php_phongo.c

```diff
@@ -3321,22 +3196,12 @@ void phongo_manager_init(php_phongo_manager_t* manager, const char* uri_string,
 	if (EG(exception)) {
 		goto cleanup;
 	}
-
-	if (!php_phongo_uri_finalize_tls(uri)) {
-		/* Exception should already have been thrown */
-		goto cleanup;
-	}
-#else
-	if (mongoc_uri_get_tls(uri)) {
-		phongo_throw_exception(PHONGO_ERROR_INVALID_ARGUMENT, "Cannot create SSL client. SSL is not enabled in this build.");
```

Comment:
`mongoc_client_new_from_uri_with_error()` now raises an error if TLS is specified (e.g. `tls=true` or implicitly through `mongodb+srv://`), so this was no longer needed.